api/auth/index.ts

```diff
@@ -165,17 +179,200 @@ export async function oauth(req: express.Request, res: express.Response, next: e
   }
 }
 
-export function doLogout(req: express.Request, res: express.Response, status: number = 302) {
+export function doLogoutOAuth2(req: express.Request, res: express.Response, status) {
   res.clearCookie(getConfigValue(COOKIE_TOKEN))
   res.clearCookie(getConfigValue(COOKIE_ROLES))
   res.clearCookie(getConfigValue(COOKIES_USERID))
   req.session.user = null
   delete req.session.auth // delete so it does not get returned to FE
   req.session.save(() => {
-    res.redirect(status, req.query.redirect || '/')
+    const redirectUrl = req.query.redirect ? req.query.redirect : '/'
```

Comment:
this const isn't being used and might break existing functionality? I would test this

Reply:
removed the unused const